noir-projects/aztec-nr/aztec/src/macros/events/mod.nr

```diff
@@ -4,8 +4,7 @@ use protocol_types::meta::generate_serialize_to_fields;
 comptime fn generate_event_interface(s: StructDefinition) -> Quoted {
     let name = s.name();
     let typ = s.as_type();
-    let (serialization_fields, _) =
-        generate_serialize_to_fields(quote { self }, typ, &[quote {self.header}], false);
+    let (serialization_fields, _) = generate_serialize_to_fields(quote { self }, typ, &[], false);
```

Comment:
We no longer need to pass the `quote {self.header}]` in field to omit here because now there is no ugly header 🥳